app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchScreen.kt

```diff
@@ -47,6 +67,12 @@ class OverdueSearchScreen : BaseScreen<
 
   private val disposable = CompositeDisposable()
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":56,"endLine":68,"prLink":"https://app.graphite.dev/github/pr/simpledotorg/simple-android/4019"}} -->
<i class='graphite__hidden'>[Re: lines 56 to 68]<br/></i>

The placing of these variable seems odd as it breaks the rest of `Inject` variables. Let's move them to `searchQueryTextChanges` variable? <span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4019?utm_source=unchanged-line-comment">Graphite</a>.</span>